packages/block-editor/src/components/use-setting/index.js

```diff
@@ -62,17 +64,20 @@ const deprecatedFlags = {
  * const isEnabled = useSetting( 'typography.dropCap' );
  * ```
  */
-export default function useSetting( path ) {
+export default function useSetting( path, name = '', store ) {
```

Comment:
Hi there! I think for me it doesn't make sense to pass a "store" here. (Why are we accessing the editSite store in the block-editor package, what happens if the shape of the edit-site store changes but not the block-editor)  It feels like we're sharing code just for the sake of doing so. In these situations, I think it's probably better to leave the duplication in place. what may seem similar may not be in reality (edit-site allows editing these but not block-editor)